cmd/virt-launcher/sh.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/sh.orig
+
+if [ ${TERM} = "xterm" ] ; then
```

Comment:
Another option would be to scan for the exact commandline invocation.

Reply:
@rmohr any chance we could get them to add something else special to the commandline that almost 100% guarantees no one could accidentally connect to the VM instead of the pod

Reply:
Would be nice to only match on the specific commandline invocation.